cf-agent/verify_files.c

```diff
@@ -639,29 +607,23 @@ static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp,
     {
         unsigned char rendered_output_digest[EVP_MAX_MD_SIZE + 1] = { 0 };
         HashString(BufferData(output_buffer), BufferSize(output_buffer), rendered_output_digest, CF_DEFAULT_DIGEST);
+
         if (!HashesMatch(existing_output_digest, rendered_output_digest, CF_DEFAULT_DIGEST))
         {
-            if (a.transaction.action == cfa_warn || DONTDO)
-            {
-                cfPS(ctx, LOG_LEVEL_WARNING, PROMISE_RESULT_WARN, pp, &a,
-                     "Need to update rendering of '%s' from mustache template '%s' but policy is dry-run",
-                     pp->promiser, message);
-                result = PromiseResultUpdate(result, PROMISE_RESULT_WARN);
+            char *d = (char *) BufferData(output_buffer), *s, *last;
```

Comment:
Please declare/define variables on separate lines.

Reply:
Also please give the variables more descriptive names.

Reply:
Will do.